### DISCUSSION TITLE
Questions about scatac and scrna joint analysis

### DISCUSSION THREAD

**Discussion by HelloWorldLTY** [stuart-lab/signac#1220] [Q&A] [answered]
Hi, sorry to disturb you again. I wonder for this notebook:
https://satijalab.org/signac/articles/pbmc_multiomic.html
Is the step for regenerating peaks, known as Peak calling, really necessary? Is it fine if I ignore this step? Thanks a lot.

In addition, are there any tools I can use to transfer h5seurat file in to h5 file for multiome data? Thanks a lot.

**Comment by timoast** [comment, marked as answer]
> Is the step for regenerating peaks, known as Peak calling, really necessary? Is it fine if I ignore this step? Thanks a lot.

No, this step is not necessary

> In addition, are there any tools I can use to transfer h5seurat file in to h5 file for multiome data? Thanks a lot.

Not sure what you mean exactly, but questions about h5seurat can be asked here: https://github.com/mojaveazure/seurat-disk
